edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/brokerConnection/MqttBrokerConnector.cs

```diff
@@ -85,6 +85,7 @@ public async Task ConnectAsync(string serverAddress, int port)
                     this.mqttClient = Option.None<MqttClient>();
                 }
 
+                Events.CouldNotConnect();
                 throw new Exception("Failed to start mqtt-bridge connector");
```

Comment:
Nit: can we change this to "Failed to start MQTT broker connector"? Once we add the bridge, this line can get a bit confusing.